biomage/stage/stage.py

```diff
@@ -248,49 +253,55 @@ def get_sandbox_id(templates, manifests, all_experiments):
-    click.echo()
-    click.echo(
-        click.style(
-            "A sandbox will be created from the manifest files listed above. "
-            "Now specify which deployments you would like to pin.",
-            fg="yellow",
-            bold=True,
+    # autopin the repos on the default branch
+    if auto:
+        pins = [
+            name for name, props in templates.items() if props.ref == DEFAULT_BRANCH
+        ]
+    else:
+        # Ask about which releases to pin.
+        click.echo()
+        click.echo(
+            click.style(
+                "A sandbox will be created from the manifest files listed above. "
+                "Now specify which deployments you would like to pin.",
+                fg="yellow",
+                bold=True,
+            )
         )
-    )
-    click.echo(
-        "The sandbox will not be affected by any future changes made to pinned "
-        "deployments. For example, if you pin `ui`,\n"
-        f"no new changes made to the {DEFAULT_BRANCH} branch of the `ui` repository "
-        "will propagate to your sandbox after it’s created.\n"
-        f"By default, only deployments sourced from the {DEFAULT_BRANCH} "
-        "are pinned, deployments using branches you are\n"
-        "likely to be testing (e.g. pull requests) are not."
-    )
-    questions = [
-        {
-            "type": "checkbox",
-            "name": "pins",
-            "message": "Which deployments would you like to pin?",
-            "choices": [
-                {"name": name, "checked": props.ref == DEFAULT_BRANCH}
-                for name, props in templates.items()
-            ],
-        }
-    ]
+        click.echo(
+            "The sandbox will not be affected by any future changes made to pinned "
+            "deployments. For example, if you pin `ui`,\n"
+            f"no new changes made to the {DEFAULT_BRANCH} branch of the `ui` repository "
+            "will propagate to your sandbox after it’s created.\n"
+            f"By default, only deployments sourced from the {DEFAULT_BRANCH} "
+            "are pinned, deployments using branches you are\n"
+            "likely to be testing (e.g. pull requests) are not."
+        )
+        questions = [
+            {
+                "type": "checkbox",
+                "name": "pins",
+                "message": "Which deployments would you like to pin?",
+                "choices": [
+                    {"name": name, "checked": props.ref == DEFAULT_BRANCH}
+                    for name, props in templates.items()
+                ],
+            }
+        ]
 
-    click.echo()
-    pins = prompt(questions)
-    try:
-        pins = set(pins["pins"])
-        click.echo("Pinned repositories:")
-        click.echo("\n".join(f"• {pin}" for pin in pins))
+        click.echo()
+        pins = prompt(questions)
+        try:
+            pins = set(pins["pins"])
 
-    except Exception:
-        exit(1)
+        except Exception:
+            exit(1)
```

Comment:
Why do we do this?

Reply:
I honestly have no idea it was there but it makes no sense afaik. I've removed it